### PR TITLE
Initial Release

### PR DESCRIPTION
**Pull request by elijahmanor**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
+report/
+bower/
+public/css
+public/js
+public/*.html
+public/images
+server/log
+_spec-tmp
+karma-index.js
+client/js/clientConfig.js
```
Comment (dcneiner) [inline .gitignore]: Maybe we could clean some of this up?

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+demo/
```
Comment (dcneiner) [inline .npmignore]: Possibly revise this as well to just list what is relevant to this repo.

File: .gitignore [hunk 3]
```diff
@@ -1,33 +1,3 @@
...
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directory
 node_modules
```
Comment (elijahmanor) [inline .gitignore]: I trimmed down the `.gitignore` file to just the things needed for this repo and I deleted the `.npmignore` file as `npm` defaults to using the `.gitignore` file if there isn't a `.npmignore` file present

> "If there's no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file." -- [Keeping files out of your package](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)

File: src/sequence-steps.js [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+	gitPushUpstreamMaster,
+	npmPublish,
+	gitCheckoutDevelop,
+	gitMergeMaster,
+	gitPushUpstreamDevelop,
+	gitPushOriginMaster
+];
+
+export function gitFetchUpstreamMaster( [ git, options ] ) {
+	const command = "git fetch upstream --tags";
```
Comment (elijahmanor) [inline src/sequence-steps.js]: This used to be "git fetch upstream master --tags", but now it is just "git fetch upstream --tags"

File: src/sequence-steps.js [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+	if ( options.develop ) {
+		utils.log.begin( command );
+		return nodefn.lift( ::git.push )( "upstream", "develop" )
+			.then( () => utils.log.end() );
+	}
+	return null;
+}
+
+export function gitPushOriginMaster( [ git, options ] ) {
+	const command = `git push origin master`;
```
Comment (elijahmanor) [inline src/sequence-steps.js]: I updated this variable to match the actual command that is taking place

**Pull request merged**